frontend/src/components/themes/DefaultTheme.scss

```diff
@@ -8,7 +8,9 @@
   --spacing-small: 1rem;
   --spacing-medium: 1.5rem;
   --spacing-large: 2rem;
-  --spacing-extra-large: 3rem;
+  --spacing-extra-large: 2.5rem;
```

Comment:
Slovo spacing se překládá jako odsazení (v typografii, ale jinak je to vzdálenost, která má taky správný význam, ale nezní to tak učeně). V CSS je proto slovo spacing spjato s paddingem a marginem. Pokud určuješ velikost, tak se používá slovo size.

Reply:
Viz výše, teď neřešme, udělám v #34 PR.

Reply:
Pisu jak to bylo zacato